src/main/java/net/minestom/server/instance/palette/PaletteIndirect.java

```diff
@@ -49,8 +49,6 @@ final class PaletteIndirect implements SpecializedPalette, Cloneable {
             this.paletteToValueList.add(palette[i]);
             this.valueToPaletteMap.put(palette[i], i);
         }
-
-        this.values = new long[arrayLength(dimension(), bitsPerEntry)];
```

Comment:
Is there some reason why that happened or how we can reproduce it?

If it possible to add a test for that to prevent such behaviour in the future.

Reply:
Well a simple test would be writing a palette into a NetworkBuffer, reading it back out, and comparing that they're the same...

I'm not sure what you're trying to reproduce here, the state of the code is as such and it makes no sense to pass values a a parameter to the constructor and then immediately override the field

Reply:
I just don't have the proof, the test would ensure that other contributors don't make the same mistake

Reply:
I described a test that would prevent this specific mistake, the general way to prevent this type of mistake is code review.

Reply:
Implement a test instead of describing it

Reply:
Does that test satisfy your requirements?